### DISCUSSION TITLE
Webxr ar-button is not shown on some pop-up pages

### DISCUSSION THREAD

**Discussion by sercandemirtas** [google/model-viewer#2416] [Q&A]
Hello friends!

As I mentioned on the tittle on some pop-up pages Webxr ar-button is not shown. When we use scene-viewer, ar-button appears on these pop-up pages which are not show Webxr ar-button.

Is it a kind of bug or privacy policy for Webxr ar-button? 

How can we solve that? Any suggestion or solution would be great.

Thanks...

### Version

-->
- model-viewer: v1.3.0

### AR

- [x ] WebXR
- [ ] SceneViewer
- [ ] QuickLook

**Comment by elalish** [comment]
With v1.7 you should get some helpful console logs explaining what happened with attempting AR. I would guess you might be missing an iframe permission or HTTPS: https://modelviewer.dev/examples/augmentedreality/#ar
